src/test/iv_client.c

```diff
@@ -65,13 +65,6 @@ static crt_endpoint_t	g_server_ep;
 static char		g_hostname[100];
```

Comment:
this likely becomes not needed either, as this was being used previously in the debug print macro